includes/Admin/Settings.php

```diff
@@ -493,20 +493,74 @@ public function get_settings_fields() {
-                    'default' => 'percentage',
+                    'default' => 'fixed',
                     'tooltip' => __( 'Select a commission type', 'dokan-lite' ),
                 ],
-                'admin_percentage'       => [
-                    'name'              => 'admin_percentage',
-                    'label'             => __( 'Admin Commission', 'dokan-lite' ),
-                    'desc'              => __( 'Amount you get from each sale', 'dokan-lite' ),
-                    'default'           => '10',
-                    'type'              => 'price',
-                    'sanitize_callback' => 'wc_format_decimal',
+                'commission_fixed_values' => [
+                    'name'    => 'commission_fixed_values',
+                    'label'   => __( 'Admin Commission', 'dokan-lite' ),
+                    'type'    => 'commission_fixed',
+                    'fields'  => [
+                        'percent_fee' => [
+                            'name'                       => 'admin_percentage',
+                            'label'                      => __( 'Percent Fee', 'dokan-lite' ),
+                            'default'                    => '10',
+                            'type'                       => 'text',
+                            'desc'                       => __( 'Amount you will get from sales in percentage (10%)', 'dokan-lite' ),
+                            'required'                   => 'yes',
+                            'sanitize_callback'          => 'wc_format_decimal',
+                            'response_sanitize_callback' => 'wc_format_decimal',
+                        ],
+                        'fixed_fee' => [
+                            'name'                       => 'additional_fee',
+                            'label'                      => __( 'Fixed Fee', 'dokan-lite' ),
+                            'default'                    => '10',
+                            'type'                       => 'text',
+                            'desc'                       => __( 'Amount you will get from sales in flat rate(+5)', 'dokan-lite' ),
+                            'required'                   => 'yes',
+                            'sanitize_callback'          => 'wc_format_decimal',
+                            'response_sanitize_callback' => 'wc_format_localized_price',
+                        ],
+                    ],
+                    'default' => 'fixed',
+                    'min'     => '0',
+                    'step'    => 'any',
+                    'desc'    => __( 'Amount you will get from sales in both percentage and fixed fee', 'dokan-lite' ),
+                    'sanitize_callback'          => 'wc_format_decimal',
+                    'response_sanitize_callback' => 'wc_format_localized_price',
+                    'show_if' => [
+                        'commission_type' => [
+                            'equal' => 'fixed',
```

Comment:
Vai we don't need to do it because we are running the upgrader and it will set the global commission type to fixed if it is saved as flat/percentage/combine. So the condition is not needed it will word fine.